src/core/components/workflow-status/udb.workflow-status.directive.js

```diff
@@ -58,6 +61,34 @@ function WorkflowStatusDirectiveController($scope, appConfig) {
     }
   }
 
+  /**
+   * show the publication url
+   * @returns {boolean}
+   */
+  function showPublicUrl () {
+    return isPlace() || cm.event.audience.audienceType === 'everyone';
+  }
+
+  /**
+   * get the url for cultuurkuur
+   * @param {string} cdbid
+   */
+  function getCultuurKuurKUrl (cdbid) {
+    if (appConfig.cultuurkuurUrl && !isPlace()) {
```

Comment:
Shouldn't the `!isPlace()` check technically be a part of `showCultuurkuurUrl()`?